tests/test_dummy_animal_module.py

```diff
@@ -0,0 +1,61 @@
+"""Test module for dummy_animal_module.py.
+
+This module tests the functionality of dummy_animal_module.py
+"""
+
+# Problems:
+# handling string returns for details
+# handling initialization of animals with zero energy
+# test that eat can't drive energy negative
+
+import virtual_rainforest.animals.dummy_animal_module as am
+
+
+class TestPlant:
+    """Test Plant class."""
+
+    def test_plant_growth0(self):
+        """Testing plant_growth at 100% energy."""
+        p = am.Plant("tree", 10.0, 1)
+        print("testing plant growth : plant")
```

Comment:
I wouldn't put `print` statements in tests or use LOGGER in them. When they fail `pytest` prints information on why they failed, but otherwise they should generally be silent.

Reply:
Agree. Tests should not print or log things directly (they might do so indirectly when testing something, but not include those as part of the code of the test)

Reply:
Interesting, thanks.